dips/dip-1.mdx

```diff
@@ -749,7 +749,7 @@ def travel_rule(
 
 
 def add_payment_recipient_signature(payment: PaymentObject) -> None:
-    """`recipient_signature` will be used as the `metadata_signature` parameter in [peer_to_peer_with_metadata script](https://github.com/diem/diem/blob/master/language/stdlib/transaction_scripts/doc/transaction_script_documentation.md#peer_to_peer_with_metadata)"""
```

Comment:
I don't think this code has any place in a DIP, but I'm happy to push on that one independently :)